gpcontrib/arenadata_toolkit/src/arenadata_toolkit_guc.c

```diff
@@ -0,0 +1,295 @@
+#include "arenadata_toolkit_guc.h"
+
+
+#include "cdb/cdbvars.h"
+#include "catalog/objectaccess.h"
+#include "catalog/pg_db_role_setting.h"
+#include <limits.h>
+#include "utils/guc.h"
+#include "tf_shmem.h"
+
+#define DEFAULT_BLOOM_SIZE 1000000
+#define DEFAULT_DB_TRACK_COUNT 5
+#define DEFAULT_IS_TRACKED false
+#define DEFAULT_DROPS_COUNT 100000
+#define DEFAULT_TRACKED_SCHEMAS "public,arenadata_toolkit,pg_catalog,pg_toast,pg_aoseg,information_schema"
+#define DEFAULT_GET_FULL_SNAPSHOT_ON_RECOVERY true
+#define DEFAULT_TRACKED_REL_STORAGES "h,a,c"
+#define DEFAULT_TRACKED_REL_KINDS "r,i,t,m,o,b,M"
+#define DEFAULT_NAPTIME 60
+
+#define MIN_BLOOM_SIZE 1
```

Comment:
Does it make sense to have bloom size of 1 byte? Maybe it is reasonable to increase the allowed min size?